tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -170,10 +170,17 @@ class HighThroughputTest(RedpandaTest):
 
     def __init__(self, test_ctx: TestContext, *args, **kwargs):
         self._ctx = test_ctx
-
         # Default set to tier-1-aws is a temporary work around for
         # https://github.com/redpanda-data/cloudv2/issues/7903
-        cloud_tier_str = test_ctx.globals.get("cloud_tier", "tier-1-aws")
+        if RedpandaServiceCloud.GLOBAL_CLOUD_CLUSTER_CONFIG in self._ctx.globals:
```

Comment:
Would it be better to get config profile name from the redpandaCloudService instance:

`self.redpanda._cloud_cluster.config.config_profile_name`

and instead check the instance type of self.redpanda is cloud rather than docker?

Reply:
At this point there is no self.redpanda created yet. And doing it like this is easier than overriding setUp method that is creating actual cluster.